pynest/examples/Potjans_2014/network_params.py

```diff
@@ -168,91 +83,90 @@ def get_std_PSP_matrix(PSP_rel, number_of_pop):
-    # 'original': uniform mean and std for all populations.
-    # 'optimized': population-specific mean and std, allowing a reduction of
-    # the initial activity burst in the network.
-    # Choose either 'original' or 'optimized'.
-    'V0_type': 'original',
-    # Parameters of the neurons.
+    # initial conditions for the membrane potential, options are:
+    # 'original': uniform mean and standard deviation for all populations as
+    #             used in earlier implementations of the model
+    # 'optimized': population-specific mean and standard deviation, allowing a
+    #              reduction of the initial activity burst in the network
+    #              (default)
+    'V0_type': 'optimized',
+    # parameters of the neuron model
     'neuron_params': {
-        # Membrane potential average for the neurons (in mV).
+        # membrane qpotential average for the neurons (in mV)
         'V0_mean': {'original': -58.0,
                     'optimized': [-68.28, -63.16, -63.33, -63.45,
                                   -63.11, -61.66, -66.72, -61.43]},
-        # Standard deviation of the average membrane potential (in mV).
-        'V0_sd': {'original': 10.0,
-                  'optimized': [5.36, 4.57, 4.74, 4.94,
-                                4.94, 4.55, 5.46, 4.48]},
-        # Reset membrane potential of the neurons (in mV).
+        # standard deviation of the average membrane potential (in mV)
+        'V0_std': {'original': 10.0,
+                   'optimized': [5.36, 4.57, 4.74, 4.94,
+                                 4.94, 4.55, 5.46, 4.48]},
+        # reset membrane potential of the neurons (in mV)
         'E_L': -65.0,
-        # Threshold potential of the neurons (in mV).
+        # threshold potential of the neurons (in mV)
         'V_th': -50.0,
-        # Membrane potential after a spike (in mV).
+        # membrane potential after a spike (in mV)
         'V_reset': -65.0,
-        # Membrane capacitance (in pF).
+        # membrane capacitance (in pF)
         'C_m': 250.0,
-        # Membrane time constant (in ms).
+        # membrane time constant (in ms)
         'tau_m': 10.0,
-        # Time constant of postsynaptic excitatory currents (in ms).
-        'tau_syn_ex': 0.5,
-        # Time constant of postsynaptic inhibitory currents (in ms).
-        'tau_syn_in': 0.5,
-        # Time constant of external postsynaptic excitatory current (in ms).
-        'tau_syn_E': 0.5,
-        # Refractory period of the neurons after a spike (in ms).
-        't_ref': 2.0}
-}
+        # time constant of postsynaptic currents (in ms)
+        'tau_syn': 0.5,
+        # refractory period of the neurons after a spike (in ms)
+        't_ref': 2.0}}
+
+# derive matrix of mean PSPs,
+# the weight of the connection from L4E to L23E is doubled
+mean_PSP_matrix = get_exc_inh_matrix(
+    net_dict['PSP_e'],
+    net_dict['PSP_e'] * net_dict['g'],
+    len(net_dict['populations']))
+mean_PSP_matrix[0, 2] *= 2
```

Comment:
If someone accidentally called this function twice, the L4E->L23E strength would be quadrupled. Normally this shouldn't happen, but by defining a variable PSP_23e_4e = 2 * PSP_e and then setting mean_PSP_matrix[0, 2] = PSP_23e_4e you avoid this altogether.

Reply:
I did not define a variable, but directly set the matrix entry now to the correct value.
```suggestion
mean_PSP_matrix[0, 2] = 2. * net_dict['PSP_e']
```
Are you fine with this?

Reply:
I changed to `PSP_matrix_mean[0, 2] = 2. * net_dict['PSP_exc_mean']`. Parameter names have also changed. Please let me know if this is fine with you.